src/styles/colour/index.md.njk

```diff
@@ -34,8 +34,12 @@ use `govuk-colour("red")` rather than `$govuk-error-colour`.
   {% for groupName, group in colours.applied %}
     <tr>
       <td colspan="3">
-        <h3 class="govuk-heading-m {% if not loop.first %}govuk-!-padding-top-6{% endif %}">
-        {{groupName}} 
+        <h3 class="app-content-heading" id="{{ groupName | slugger }}">
```

Comment:
I wonder if I could make a Nunjucks partial and share them between the renderer and here...